app/src/common/shared/com/igalia/wolvic/browser/PromptDelegate.java

```diff
@@ -533,4 +551,4 @@ public void onSessionChanged(@NonNull Session aOldSession, @NonNull Session aSes
         cleanSession(aOldSession);
         setUpSession(aSession);
     }
-}
+}
```

Comment:
This line change is not needed.

Reply:
For this line, I think the code added to the file above has shifted the last brace.
No changes from my end are fixing it.